mkdocs.yml

```diff
@@ -95,6 +95,7 @@ nav:
         - examples/consul.md
         - examples/nginx.md
         - examples/toxiproxy.md
+        - examples/postgres.md
```

Comment:
Let's remove this from the docs site, as we want to have everything in module page
```suggestion
```